src/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/sdk:5.0-alpine AS build
+FROM mcr.microsoft.com/dotnet/sdk:6.0.102-alpine3.14 AS build
```

Comment:
Is there a reason why we are pinning the alpine version now?

Reply:
Good question, don't think so. @TheTechArch did you experience issues when you did this?

Reply:
Changed to use latest.